src/Routing/RequestActionTrait.php

```diff
@@ -97,6 +98,8 @@
      */
     public function requestAction($url, array $extra = [])
     {
+        trigger_error('requestAction() is deprecated. Use view cells instead.', E_USER_DEPRECATED);
```

Comment:
I'm not sure we should emit runtime errors just yet. I was thinking we'd do a release before 4.0 where all documented deprecations would start emitting errors. Doing documentation only warnings lets us 'soft fail' people and allows them to opt-in to hard failures later on.